platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -145,14 +145,16 @@ class ONIEUpdater(object):
     # For SN2201, upgrading fireware from ONIE is supported from day one so we do not need to check it.
     PLATFORM_ALWAYS_SUPPORT_UPGRADE = ['x86_64-nvidia_sn2201-r0']
 
-    BIOS_UPDATE_FILE_EXT = '.rom'
-    
+    BIOS_UPDATE_FILE_EXT_ROM = '.rom'
+    BIOS_UPDATE_FILE_EXT_CAB = '.CAB'
```

Comment:
@DavidZagury do we really need to stick to all capital in extension?

Reply:
Fixed